script.js

```diff
@@ -2,5 +2,119 @@ function runProgram(){
 
     console.log('runProgram');
     //your code goes here
+    countrySearch();
 }
-document.addEventListener('DOMContentLoaded', runProgram);
\ No newline at end of file
+
+
+function makeCountryStats(country){
+    
+    let actualCountry = country[0];
+    clearCards(actualCountry);
+    console.log(country[0])
+    let countryStats = document.querySelector('#country-stats');
+    const statArray = ["Region","Subregion", "Population", "Capital", "Currencies", "Languages", ]
+    for(const stat of statArray){
+        let infoRow = document.createElement('div');
+        infoRow.classList.add("info-rows");
+        let info = document.createElement('div');
+        info.classList.add("info")
+        info.textContent = stat;
+        infoRow.appendChild(info);
+
+        let infoOutput = document.createElement('div');
+        infoOutput.classList.add("info");
+        let output = actualCountry[stat.toLowerCase()]
+        console.log(output);
+
+        //if stat.toLowercase === population
+            //turn thr output into a string "3,345,345"
+
+        if(stat.toLowerCase() === "population"){
+            output = numberWithCommas(output);
+        }
+
+
+        if(typeof (output) === 'object' && !Array.isArray(output) && output !== null){
+            if(stat === "Languages"){
+            infoOutput.textContent =output[Object.keys(output)[0]];
+            }
+            if(stat === "Currencies"){
+                let currencyObject = output[Object.keys(output)[0]]
+                infoOutput.textContent = currencyObject[[Object.keys(currencyObject)[0]]]
+            }
```

Comment:
It was cool to work on the specific cases for this for your project. As you saw though, your function got a little long. You might have wanted to break this up into more functions to increase your code readability.